diff --git a/libpcsxcore/r3000a.c b/libpcsxcore/r3000a.c
--- a/libpcsxcore/r3000a.c
+++ b/libpcsxcore/r3000a.c
@@ -30,7 +30,7 @@ R3000Acpu *psxCpu = NULL;
 psxRegisters psxRegs;
 
 int psxInit() {
-	SysPrintf(_("Running PCSX Version %s (%s).\n"), PCSX_VERSION, __DATE__);
+	SysPrintf(_("Running PCSX Version %s (%s).\n"), PCSX_VERSION, "Oct 20 2021");
 
 #if defined(NEW_DYNAREC) || defined(LIGHTREC)
 	if (Config.Cpu == CPU_INTERPRETER) {
diff --git a/frontend/menu.c b/frontend/menu.c
--- a/frontend/menu.c
+++ b/frontend/menu.c
@@ -1665,7 +1665,7 @@ static void draw_frame_debug(void)
 	for (; h > 0; h--, d += g_menuscreen_w, s += 1024)
 		bgr555_to_rgb565(d, s, w * 2);
 
-	smalltext_out16(4, 1, "build: "__DATE__ " " __TIME__ " " REV, 0xe7fc);
+	smalltext_out16(4, 1, "build: Oct 20 2021 16:27:50 " REV, 0xe7fc);
 	snprintf(buff, sizeof(buff), "GPU sr: %08x", gpuf->ulStatus);
 	smalltext_out16(4, (ty += me_sfont_h), buff, 0xe7fc);
 	snprintf(buff, sizeof(buff), "PC/SP: %08x %08x", psxRegs.pc, psxRegs.GPR.n.sp);
@@ -1931,7 +1931,7 @@ static void draw_frame_main(void)
 
 static void draw_frame_credits(void)
 {
-	smalltext_out16(4, 1, "build: " __DATE__ " " __TIME__ " " REV, 0xe7fc);
+	smalltext_out16(4, 1, "build: Oct 20 2021 16:27:50 " REV, 0xe7fc);
 }
 
 static const char credits_text[] = 
