diff --git a/frontend/main.c b/frontend/main.c
--- a/frontend/main.c
+++ b/frontend/main.c
@@ -454,6 +454,6 @@ int emu_core_init(void)
 {
 	SysPrintf("Starting PCSX-ReARMed " REV " (%s)\n", get_build_info());
-	SysPrintf("build time: " __DATE__ " " __TIME__ "\n");
+	SysPrintf("build time: Oct 20 2021 16:27:50\n");
 
 #if defined(__arm__) && defined(__ARM_FP)
 	// RunFast mode
diff --git a/frontend/menu.c b/frontend/menu.c
--- a/frontend/menu.c
+++ b/frontend/menu.c
@@ -2138,7 +2138,7 @@ static void draw_frame_main(void)
 
 static void draw_frame_credits(void)
 {
-	smalltext_out16(4, 1, "build: " __DATE__ " " __TIME__ " " REV, 0xe7fc);
+	smalltext_out16(4, 1, "build: Oct 20 2021 16:27:50 " REV, 0xe7fc);
 }
 
 static const char credits_text[] = 
